### PR TITLE
Add XYZprinting materials

### PR DESCRIPTION
**Pull request by heed818**
Add XYZprinting materials

### CODE REVIEW

File: xyzprinting_metallic_pla.xml.fdm_material [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
+      <material>Premium Metallic PLA</material>
```
Comment (nallath) [inline xyzprinting_metallic_pla.xml.fdm_material:6]: The material tag is about the material _type_. So the type of this material is PLA, not Premium Metallic PLA.

File: xyzprinting_flexible.xml.fdm_material [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
+      <material>Flexible</material>
```
Comment (nallath) [inline xyzprinting_flexible.xml.fdm_material:6]: What actual material is this? TPU?

File: xyzprinting_antibact_pla.xml.fdm_material [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
+      <material>Anti-Bact</material>
```
Comment (nallath) [inline xyzprinting_antibact_pla.xml.fdm_material:6]: What type should this be? Anti-bat isn't a plastic that I know ;)

File: xyzprinting_tough_pla.xml.fdm_material [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
```
Comment (nallath) [inline xyzprinting_tough_pla.xml.fdm_material:5]: If this isn't different from generic tough PLA, there isn't really a need to add this.

**Review by nallath — CHANGES_REQUESTED** [review]

File: xyzprinting_metallic_pla.xml.fdm_material (hunk 1 above)
Comment (heed818) [reply xyzprinting_metallic_pla.xml.fdm_material:6]: OK~thank you :)

File: xyzprinting_flexible.xml.fdm_material (hunk 2 above)
Comment (heed818) [reply xyzprinting_flexible.xml.fdm_material:6]: It is TPEE,i will update this file,thank you :)

File: xyzprinting_antibact_pla.xml.fdm_material (hunk 3 above)
Comment (heed818) [reply xyzprinting_antibact_pla.xml.fdm_material:6]: It is PLA, Anti-Bacteria PLA
Thank you :)

File: xyzprinting_tough_pla.xml.fdm_material (hunk 4 above)
Comment (heed818) [reply xyzprinting_tough_pla.xml.fdm_material:5]: ok, i got it, but the temperature is a little different.

**Comment by heed818** [comment]
Hi @Ghostkeeper 
i update the description and color for the material you mentioned,thank you.

**Review by Ghostkeeper — APPROVED** [review]
Looks good now. I'll give these a test and if everything is working we'll merge it for 4.13.

**Comment by heed818** [comment]
> Looks good now. I'll give these a test and if everything is working we'll merge it for 4.13.

Thank you,Sir

**Pull request merged**